corehq/apps/es/mappings/domain_mapping.py

```diff
@@ -429,14 +429,14 @@
             "type": "boolean"
         },
         "is_test": {
-            "type": "text"
+            "type": "keyword"
```

Comment:
Ha, was gonna ask if this is a boolean, but I just checked and it's not:
https://github.com/dimagi/commcare-hq/blob/b299e00d95a7bfd6531f99e2d4b9941bae594e96/corehq/apps/domain/models.py#L320-L320